iree/compiler/Dialect/VM/Conversion/VMToEmitC/test/arithmetic_ops.mlir

```diff
@@ -1,12 +1,10 @@
 // RUN: iree-opt -split-input-file -pass-pipeline='vm.module(iree-convert-vm-to-emitc)' %s | IreeFileCheck %s
 
-// CHECK: vm.module @my_module {
+// CHECK-LABEL: @add_i32
```

Comment:
In general it's a good idea to match on the `vm.func ` (or whatever type the symbol you want to match is) - with just the symbol name *any* use of the symbol name will match and it leads to really annoying errors. For example, `vm.export @add_i32` will match as well as `vm.func @add_i32`, etc. It's only because vm.export is added after vm.func this works right now, which is brittle. I've encountered this a bunch higher up in the stack (including just yesterday) - it sucks to debug :)

(doesn't have to be cleaned up in this PR, but it's a good habit to get into)

Reply:
Uh, actually I have adopted this to be consistent with `iree/compiler/Dialect/VM/IR/test/arithmetic_ops.mlir`. But I see, we need a cleanup of the cleanup ;)